hack/charts/namespaced/operator_chart/templates/NOTES.txt

```diff
@@ -0,0 +1,10 @@
+Amazon SageMaker Operator for Kubernetes installed!
```

Comment:
nit: Does not specify this is the operator

Reply:
I think this is the cleanest way of writing it, have not changed this.